package.json

```diff
@@ -52,7 +52,7 @@
     }
   },
   "lint-staged": {
-    "*.js": [
+    "packages/!(fxa-payments-server)/**/*.js": [
```

Comment:
Payments is failing eslint.  There's no quick fix other than this at the moment.

Reply:
Yes, it's https://mozilla-hub.atlassian.net/browse/FXA-6390